llvm/lib/ExecutionEngine/JITLink/EHFrameSupportImpl.h

```diff
@@ -21,13 +21,13 @@
 namespace llvm {
 namespace jitlink {
 
-/// A LinkGraph pass that splits blocks in an eh-frame section into sub-blocks
-/// representing individual eh-frames.
-/// EHFrameSplitter should not be run without EHFrameEdgeFixer, which is
-/// responsible for adding FDE-to-CIE edges.
-class SectionSplitter {
+/// A LinkGraph pass that splits blocks in a section that follows the DWARF
+/// Record format into sub-blocks where each header gets it's own block.
```

Comment:
Should be "its", rather than "it's". Not a blocker -- can be fixed up before landing or in a follow-up.